Tasks/JavaToolInstallerV0/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "java-tool-installer",
-    "version": "0.211.0",
+    "version": "0.216.0",
```

Comment:
this version change has no effect on anything.
I would suggest removing it and making the package private
https://docs.npmjs.com/cli/v9/configuring-npm/package-json#private